documentation/examples/08-motion/01-spring/App.svelte

```diff
@@ -28,6 +28,7 @@
 	on:mousemove={(e) => coords.set({ x: e.clientX, y: e.clientY })}
 	on:mousedown={() => size.set(30)}
 	on:mouseup={() => size.set(10)}
+	role='button' tabindex='0'
```

Comment:
We should ignore this warning for now - this tutorial is very reliant on mouse interactions, adding role="button" won't address the root issue